src/App.js

```diff
@@ -1,11 +1,21 @@
 import React from "react";
+import { Switch, Route } from "react-router-dom";
+import Navigation from "./Components/Navigation";
+import Home from "./Components/Home";
+import Form from "./Components/Form";
 
 const App = () => {
   return (
-    <>
-      <h1>Lambda Eats</h1>
-      <p>You can remove this code and create your own header</p>
-    </>
+    <div className="App">
+    <Navigation/>
+    
+    <Switch>
+      <Route path="/order-form" render={() => <Form/>}/>
```

Comment:
a newer syntax that is popular with react-router is:
`<Route path='/order-form'><Form /></Route>`

this will make it easier to pass props into the component, it's also a little easier to read.